pkg/machine/applehv/stubber.go

```diff
@@ -7,12 +7,15 @@ import (
 	"errors"
 	"fmt"
 	"net"
+	"os"
+	"runtime"
 	"strconv"
 	"time"
 
 	"github.com/containers/common/pkg/config"
 	"github.com/containers/common/pkg/strongunits"
 	gvproxy "github.com/containers/gvisor-tap-vsock/pkg/types"
+	"github.com/containers/podman/v5/cmd/podman/registry"
```

Comment:
Do not import something from cmd/... in pkg/...

Reply:
fixed

Reply:
the current diff still shows the import

Reply:
@Luap99 Fixed. Please check again.